index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>판다마켓</title>
+
+    <!-- SEO 메타 태그 -->
+    <meta name="description" content="판다마켓에서 다양한 상품을 만나보세요. 고객님의 편리한 쇼핑을 위해 최선을 다하겠습니다.">
+    <meta name="keywords" content="판다마켓, 쇼핑, 온라인 쇼핑, 다양한 상품">
+    <meta name="author" content="판다마켓">
+
+    <!-- Open Graph (OG) 메타 태그 -->
+    <meta property="og:title" content="판다마켓 - 온라인 쇼핑의 시작">
+    <meta property="og:description" content="판다마켓에서 다양한 상품을 만나보세요. 고객님의 편리한 쇼핑을 위해 최선을 다하겠습니다.">
+    <meta property="og:type" content="website">
+
+    <!-- 트위터 카드 메타 태그 (Twitter에서 미리보기) -->
+    <meta name="twitter:card" content="summary_large_image">
+    <meta name="twitter:title" content="판다마켓 - 온라인 쇼핑의 시작">
+    <meta name="twitter:description" content="판다마켓에서 다양한 상품을 만나보세요. 고객님의 편리한 쇼핑을 위해 최선을 다하겠습니다.">
+    
+    <link rel="stylesheet" as="style" crossorigin href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.8/dist/web/static/pretendard-dynamic-subset.css" />
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="reset.css">
+</head>
+
+<body>
+    <header id="header">
+        <div class="wrap-header">
+            <img src="./img/logo.png" alt="판다마켓 로고" />
+            <a href="/login">로그인</a>
+        </div>
+    </header>
+    <main>
+        <section id="home">
+            <div class="wrap-home">
+                <div class="title-home">
+                    <h2>일상의 모든 물건을<br />거래해 보세요</h2>
+                    <button>구경하러 가기</button>
+                </div>
+                <div class="img-home">
+                    <img src="./img/Img_home_top.png" />
```

Comment:
❗️ 수정요청
아마 이미지명을 바꾸시고 html 은 안 바꾸셨나봐요~ 
```suggestion
                    <img src="./img/img_home_top.png" />
```